docs/modules/operation/pages/deep-dive/device-config-backup/introduction.adoc

```diff
@@ -1,5 +1,6 @@
 
 = Device Configuration Backup
+:description: Overview of how to back up configuration for devices like routers, firewalls, and switches with OpenNMS Horizon/Meridian, for better disaster recovery.
```

Comment:
Can also be utilized for "change management", if we're looking to enhance page metadata